source/_posts/2021-08-04-home-energy-management.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Energy Management in Home Assistant"
+description: "Monitor your energy usage, transition to sustainable energy and save money."
+date: 2021-08-04 00:01:00
+date_formatted: "August 4, 2021"
+author: Paulus Schoutsen
+author_twitter: balloob
+comments: true
+categories:
+- Release-Notes
+- Core
+og_image: /images/blog/2021-08-energy/social.png
+feedback: true
```

Comment:
Feedback on a blogpost?
We have comments on, is that not enough?

Reply:
It actually doesn't work :)

Reply:
Good! :smile: